src/query/storages/fuse/fuse/src/fuse_table.rs

```diff
@@ -118,10 +118,8 @@ impl FuseTable {
         operator = operator.layer(StorageMetricsLayer::new(data_metrics.clone()));
         // If cache op is valid, layered with ContentCacheLayer.
         if let Some(cache_op) = CacheOperator::instance() {
-            operator = operator.layer(ContentCacheLayer::new(
-                cache_op,
-                ContentCacheStrategy::Fixed(1024 * 1024),
-            ));
+            operator =
+                operator.layer(CacheLayer::new(cache_op).with_policy(FuseCachePolicy::new()));
```

Comment:
If the cache is not set:`export STORAGE_CACHE_TYPE=none` (default), does FuseCachePolicy still works?

Reply:
> If the cache is not set:`export STORAGE_CACHE_TYPE=none` (default), does FuseCachePolicy still works?

No, it won't work.